### PR TITLE
Updating load balancing instructions

### PR DESCRIPTION
**Pull request by andredewes**
## Does this introduce a breaking change?

When developers merge from main and run the server, azd up, or azd deploy, will this produce an error?
If you're not sure, try it out on an old environment.

```
[ ] Yes
[x] No
```

## Does this require changes to learn.microsoft.com docs?

This repository is referenced by [this tutorial](https://learn.microsoft.com/azure/developer/python/get-started-app-chat-template)
which includes deployment, settings and usage instructions. If text or screenshot need to change in the tutorial,
check the box below and notify the tutorial author. A Microsoft employee can do this for you if you're an external contributor.

```
[ ] Yes
[x] No
```

## Type of change

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## Code quality checklist

See [CONTRIBUTING.md](https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/CONTRIBUTING.md#submit-pr) for more details.

- [x ] The current tests all pass (`python -m pytest`).
- [ ] I added tests that prove my fix is effective or that my feature works
- [ ] I ran `python -m pytest --cov` to verify 100% coverage of added lines
- [ ] I ran `python -m mypy` to check for type errors
- [ ] I either used the pre-commit hooks or ran `ruff` and `black` manually on my code.

### CODE REVIEW

File: docs/productionizing.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ If the maximum TPM isn't enough for your expected load, you have a few options:
...
 
-* If you are consistently going over the TPM, then consider implementing a load balancer between OpenAI instances. Most developers implement that using Azure API Management using [the openai-apim-lb repo](https://github.com/Azure-Samples/openai-apim-lb) or with Azure Container Apps using [the openai-aca-lb repo](https://github.com/Azure-Samples/openai-aca-lb). Another approach is to use [LiteLLM's load balancer](https://docs.litellm.ai/docs/providers/azure#azure-api-load-balancing) with Azure Cache for Redis.
+* If you are consistently going over the TPM, then consider implementing a load balancer between OpenAI instances. Most developers implement that using Azure API Management or container-based load balancers. For seamless integration instructions with this sample, please check:
+  * [Scale Azure OpenAI for Python with Azure API Management](https://learn.microsoft.com/en-us/azure/developer/python/get-started-app-chat-scaling-with-azure-api-management)
```
Comment (pamelafox) [inline docs/productionizing.md:28]:
```suggestion
  * [Scale Azure OpenAI for Python with Azure API Management](https://learn.microsoft.com/azure/developer/python/get-started-app-chat-scaling-with-azure-api-management)
```

File: docs/productionizing.md [hunk 2]
```diff
@@ -24,7 +24,10 @@ If the maximum TPM isn't enough for your expected load, you have a few options:
...
+  * [Scale Azure OpenAI for Python chat using RAG with Azure Container Apps](https://learn.microsoft.com/en-us/azure/developer/python/get-started-app-chat-scaling-with-azure-container-apps)
```
Comment (pamelafox) [inline docs/productionizing.md:29]:
```suggestion
  * [Scale Azure OpenAI for Python chat using RAG with Azure Container Apps](https://learn.microsoft.com/azure/developer/python/get-started-app-chat-scaling-with-azure-container-apps)
```

**Comment by pamelafox** [comment]
@john0isaac Do you know why it thinks thats a broken URL? It seems fine.

**Comment by john0isaac** [comment]
@pamelafox the only reason it would do that is if the url didn't respond with a 200 status code.
I will test it and increase the timeout for the request if needed.

Feel free to trigger the workflow again I think it will pass.

**Comment by pamelafox** [comment]
It likely gives a 300 level redirect of some sort, and then a 200 eventually, since thats our tinyURL shortener.

**Comment by john0isaac** [comment]
Yeah I though about that too because it's a redirect but the package was tested with aka.ms links and it works fine.

**Review by pamelafox — APPROVED** [review]

**Pull request merged**